apps/designer/app/shared/dom-hooks/use-measure.ts

```diff
@@ -23,7 +23,7 @@ export const useMeasure = <
   const [isInline, setIsInline] = useState(false);
 
   const handleChange = useCallback(() => {
-    if (element === null) return;
+    if (element === null || typeof window === "undefined") return;
```

Comment:
how can element be not null on the server? I mean there is no way it is being set serverside lol

Reply:
Yeah, was just trying anything when I was getting the confusing error